docs/pages/resources.md

```diff
@@ -49,6 +49,7 @@ Use a SaaS service as a backend for functionality on your Jekyll site
   - [Snipcart](https://snipcart.com/blog/static-site-e-commerce-part-2-integrating-snipcart-with-jekyll): Add a shopping cart to a Jekyll site
 
 ### Forms
+  - [Arengu] (https://www.arengu.com)
```

Comment:
```suggestion
  - [Arengu](https://www.arengu.com)
```

Whitespace is invalid so I removed. Check the markdown view to see how it looks with whitespace

https://github.com/jekyll/jekyll/blob/7356006d7019dbcd038212bd18a2d83d7f7f11cd/docs/pages/resources.md#forms